src/libtoast/src/toast_tod_filter.cpp

```diff
@@ -378,7 +378,7 @@ void toast::filter_poly2D_solve(
         int rank;
         int info;
         int one = 1;
-        double rcond_limit = 1e-3;
+        double rcond_limit = 1e-6;
```

Comment:
Eventually we could make this a parameter that is passed down from the python layer.